packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -298,7 +300,24 @@ export default class PerformanceMonitoring {
     } else if (event === INVOKE) {
       const data = task.data
       if (data && data.span) {
-        transactionService.endSpan(data.span, data)
+        const { span, response, target } = data
+        let status
+        if (response) {
+          status = response.status
+        } else {
+          status = target.status
+        }
+
+        let outcome = OUTCOME_SUCCESS
+        if (status >= 400 || status == 0) {
```

Comment:
It's >= 400 for spans (includes client errors) and >= 500 for transactions (does not include client errors)

We have to account this before setting outcome for transactions.

Reply:
That's only for backend transactions which respond to a request not for outgoing transactions.